<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VitaWidget</class>
 <widget class="QWidget" name="VitaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>58</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VitaWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="idx_label">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Цвят:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="c_label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Нюанс:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QComboBox" name="index">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>66</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A3,5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D4</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="chroma_hue">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>66</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="radioClassic">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Vita Classic</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioMaster">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Vita 3D Master</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
